File: Data Structures & Algorithms/sort-colors/submission-0.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        int n=1;
        while(n<nums.size()){
            for(int i=0;i<nums.size()-1;i++){
           if(nums[i]>nums[i+1]){
            int temp=nums[i];
            nums[i]=nums[i+1];
            nums[i+1]=temp;
           }
        } 
        n++;
        }
        
        }
};

File: Data Structures & Algorithms/majority-element-ii/submission-0.py
class Solution:
    def majorityElement(self, nums: List[int]) -> List[int]:
        n=len(nums)
        hm={}
        res=[]
        for i in nums:
            hm[i]=hm.get(i,0)+1
        
        for ele,val in hm.items():
            if val>(n//3):
                res.append(ele)

        return res